PluginCore/PluginCore/Helpers/FileHelper.cs

```diff
@@ -81,7 +81,8 @@ public static void WriteFile(String file, String text, Encoding encoding, Boolea
         {
             Boolean useSkipBomWriter = (encoding == Encoding.UTF8 && !saveBOM);
             if (encoding == Encoding.UTF7) encoding = new UTF7EncodingFixed();
-            using (StreamWriter sw = useSkipBomWriter ? new StreamWriter(file, false) : new StreamWriter(file, false, encoding))
+            using (FileStream fs = new FileStream(file, FileMode.Open))
```

Comment:
Yes, since `StreamWriter` with string did create a new file if the file doesn't exist.
> **StreamWriter Constructor (String, Boolean)**
>.NET Framework 3.5 Other Versions 
>Initializes a new instance of the StreamWriter class for the specified file on the specified path, using the default encoding and buffer size. If the file exists, it can be either overwritten or appended to. If the file does not exist, this constructor creates a new file.
Will fix.